### DISCUSSION TITLE
Mastodon link previews and Pre-rendered sites on Netlify

### DISCUSSION THREAD

**Discussion by simonv3** [mastodon/mastodon#28739] [Q&A] [answered]
Hi!

Links to our site render like this 
![afbeelding](https://github.com/mastodon/mastodon/assets/485583/bc9b624a-23b5-47ca-a390-d7eeae7f2588)
On Mastodon, 

while on eg. discord they render properly:
![afbeelding](https://github.com/mastodon/mastodon/assets/485583/a6f4fa0e-3be0-4a70-b3d2-5fb7e8bc8add)

This is done by implementing OpenGraph tags in our site <head>

Our website is an SPA that uses [netlify's prerendering](https://answers.netlify.com/t/support-guide-understanding-and-debugging-prerendering/150) to generate previews for other sites. However, netlify only renders these previews for specific user agents (eg. [list here](https://answers.netlify.com/t/support-guide-understanding-and-debugging-prerendering/150/115). 

This feels like it's likely why Mastodon isn't rendering a preview of our site. I'm assuming that's not happening in the UI? Is there a user agent we could get netlify to add to their process for this to work? If so, where's that defined in the code? 

Thanks!

**Comment by ThisIsMissEm** [comment, marked as answer]
You'd need to ask the Netlify team to add Mastodon to the pre-render user-agents list

**Reply by simonv3** [reply]
Yeah, that's where I landed. We submitted the request to Netlify here a while ago but they have yet to reply https://answers.netlify.com/t/support-guide-understanding-and-debugging-prerendering/150/136
